File: scratch/check_school.mjs
import { createClient } from '@supabase/supabase-js'

const SUPABASE_URL = 'https://supabase.campus-groovelab.de';
const SERVICE_KEY = process.env.SUPABASE_SERVICE_ROLE_KEY;

const supabase = createClient(SUPABASE_URL, SERVICE_KEY);

async function checkSchool() {
  const schoolId = "74713df2-6176-4a41-a8cd-9fbebe34e9b8";

  const { data: school, error } = await supabase
    .from('schools')
    .select('*')
    .eq('id', schoolId)
    .single();

  console.log("School error:", error);
  console.log("School details:", JSON.stringify(school, null, 2));
}

checkSchool();

File: scratch/check_school.py
import json
import urllib.request

def check_school():
    url = "https://supabase.campus-groovelab.de/rest/v1/schools?id=eq.cc05137f-5904-4774-80be-6a172c52bf99"
    service_key = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiwiaXNzIjoic3VwYWJhc2UiLCJpYXQiOjE3ODA0MTc4MTUsImV4cCI6NDkzNDAxNzgxNX0.XZd32Y-4LqKhZjiz1l-Ap6TsUk07_SEUA1QN2ot-qys"
    
    headers = {
        "apikey": service_key,
        "Authorization": f"Bearer {service_key}"
    }
    
    req = urllib.request.Request(url, headers=headers)
    try:
        with urllib.request.urlopen(req) as response:
            data = json.loads(response.read().decode())
            print(json.dumps(data, indent=2))
    except Exception as e:
        print("Error:", e)

if __name__ == "__main__":
    check_school()
